src/sagas/registry.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import { put, call, takeLatest } from "redux-saga/effects";
+import type { Saga } from "redux-saga";
+import {
+  request,
+  requestError,
+  requestSuccess,
+  type CreateProfile
+} from "../actions/registry";
+import { REGISTRY } from "../constants/actionTypes";
+import { createProfile } from "../api";
+
+export function* register(action: CreateProfile): Saga<void> {
+  try {
+    yield put(request());
```

Comment:
What the request? 
Haven't you already started this request?

Give good naming to your action creators.
Current naming is not readable at all.

Reply:
Example of good naming:
`
export const createDriver = (carrierId, driver) => ({
  type: CREATE_DRIVER,
  payload: { carrierId, driver },
});

export const createDriverSuccess = driver => ({
  type: CREATE_DRIVER_SUCCESS,
  payload: { driver },
});

`